### PR TITLE
Put an array of all the color names in ColorUtils

### PR DESCRIPTION
**Pull request by marcelgerber**
So we (and extension developers) can reuse them whereever we want.
The actual regex is unchanged, it's just getting generated now.

### CODE REVIEW

**Comment by le717** [comment]
I like this method better than stuffing them in the regex, but I'll need to check what changes I need to make to the Inline Color Editor to use this. :+1:

File: src/utils/ColorUtils.js [hunk 1]
```diff
@@ -32,13 +32,21 @@ define(function (require, exports, module) {
...
...green", "greenyellow", "grey", "honeydew", "hotpink", "indianred", "indigo", "ivory", "khaki", "lavender", "lavenderblush", "lawngreen", "lemonchiffon", "lightblue", "lightcoral", "lightcyan", "lightgoldenrodyellow", "lightgray", "lightgreen", "lightgrey", "lightpink", "lightsalmon", "lightseagreen", "lightskyblue", "lightslategray", "lightslategrey", "lightsteelblue", "lightyellow", "lime", "limegreen", "linen", "magenta", "maroon", "mediumaquamarine", "mediumblue", "mediumorchid", "mediumpurple", "mediumseagreen", "mediumslateblue", "mediumspringgreen", "mediumturquoise", "mediumvioletred", "midnightblue", "mintcream", "mistyrose", "moccasin", "navajowhite", "navy", "oldlace", "olive", "olivedrab", "orange", "orangered", "orchid", "palegoldenrod", "palegreen", "paleturquoise", "palevioletred", "papayawhip", "peachpuff", "peru", "pink", "plum", "powderblue", "purple", "red", "rosybrown", "royalblue", "saddlebrown", "salmon", "sandybrown", "seagreen", "seashell", "sienna", "silver", "skyblue", "slateblue", "slategray", "slategrey", "snow", "springgreen", "steelblue", "tan", "teal", "thistle", "tomato", "turquoise", "violet", "wheat", "white", "whitesmoke", "yellow", "yellowgreen"];
```
Comment (le717) [inline src/utils/ColorUtils.js]: Just in case this is accepted, can you go ahead and add `rebeccapurple` to this array (#10250)?

**Comment by MarcelGerber** [comment]
They are still in the regex, so you don't have to change anything.

**Comment by redmunds** [comment]
This looks good. I agree that `rebeccapurple` should be added to the list.

**Comment by le717** [comment]
@redmunds I will close #10252 in favor of this.

**Comment by MarcelGerber** [comment]
I think we need #9596 first for rebeccapurple to work properly, right.
I don't know what happens right now if you try to open an color inline editor on rebeccapurple, though.

**Comment by MarcelGerber** [comment]
Added support for `rebeccapurple` (and changed comments in order to match the color names included).
Still, the Color Inline Editor doesn't work too well with `rebeccapurple` (see https://github.com/adobe/brackets/issues/10250#issuecomment-67748764) (Quick View works perfectly fine, btw), but imho we can still merge it as there won't be many out there wanting to edit `rebeccapurple` anyway, I guess.

Sooner or later, we should, of course, merge #9596 for full support.

**Comment by le717** [comment]
@MarcelGerber I am going to try to finish up #9596 today.

**Comment by le717** [comment]
@redmunds While you're online, can you look into merging this too so we can finish up `rebeccapurple` support?

File: src/utils/ColorUtils.js [hunk 2]
```diff
@@ -32,13 +32,23 @@ define(function (require, exports, module) {
     "use strict";
     
     /**
+     * Sorted array of all the color names in the CSS Color Module Level 3 (http://www.w3.org/TR/css3-color/)
+     * and "rebeccapurple" from CSS Color Module Level 4
```
Comment (le717) [inline src/utils/ColorUtils.js]: Minor nit, does not totally need fixing: I'd reword this to 

> Sorted array of all the color names in the CSS Color Module Level 3 and 4 (http://www.w3.org/TR/css3-color/).

I don't think we need to specifically state what color(s) from CSS4 is supported, as ideally we'd support all of them.

**Comment by redmunds** [comment]
Looks good. Merging.

File: src/utils/ColorUtils.js (hunk 2 above)
Comment (MarcelGerber) [inline src/utils/ColorUtils.js]: I don't think Module Level 4 is ready yet, while Level 3 is.
While it's still in development, we should state which names are available.
